git-multimail/README

```diff
@@ -269,6 +269,25 @@ multimailhook.mailer
            The sender address to be passed to the SMTP server.  If
            unset, then the value of multimailhook.from is used.
 
+       multimailhook.smtpServerTimeout
+
+           Timeout in seconds.
+
+       multimailhook.smtpServerDebug
+
+           True/False. Set True to see all debug messages to/from
+           SMTP server.
+
+       multimailhook.smtpServerSecurity
+
+           Set the security type. Allowed values: none, ssl.
+           Default=none.
```

Comment:
The port number can be given in config option smtpServer, e.g. "mail.example.com:25".